packages/js/plugins/http/tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
extend from base config instead of duplicating

Reply:
@mpetrunic I am not sure on what base config do you think. I could extend from `js/core` ? , with something like this inside _tsconfig.json_:
```json
{
  "extends": "./../core/tsconfig",
}
```

Reply:
From root of repository

Reply:
oh heck, yeah. It should probably be separate issue

Reply:
I opened a separate issue for this #116